backend/ee/danswer/external_permissions/permission_sync.py

```diff
@@ -44,7 +44,7 @@ def run_external_group_permission_sync(
         # update postgres
         db_session.commit()
     except Exception as e:
-        logger.error(f"Error updating document index: {e}")
+        logger.exception(f"Error Syncing Group Permissions: {e}")
```

Comment:
we can remove the {e} part, since the full exception will be printed out automatically

Reply:
done